### PR TITLE
{ApplicationGateway} Include the possible value for Operator and Vari…

### PR DESCRIPTION
**Pull request by navba-MSFT**
…able parameter

On this page:
https://docs.microsoft.com/en-us/powershell/module/az.network/new-azapplicationgatewayfirewallexclusionconfig?view=azps-7.5.0

It seems it would be useful to document the valid values for the -Variable and -Operator parameters

Fixes Azure/azure-powershell#18106

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by navba-MSFT** [comment]
@MikhailTryakhov Could you please approve and merge this once you get a chance ? Thanks in advance.

**Review by VeryEarly — APPROVED** [review]

**Comment by navba-MSFT** [comment]
@VeryEarly Thanks for approving the request. Once the Tests succeeds, could you please merge this PR ?

**Pull request merged**
